moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryAutocomplete.js

```diff
@@ -1,27 +1,19 @@
 import React, { useState } from "react";
-import { Box, Grid, Icon, TextField, Typography } from "@material-ui/core";
-
+import { Grid, Box, Typography, Icon, TextField } from "@material-ui/core";
 import ProjectSummaryLabel from "./ProjectSummaryLabel";
-
 import { Autocomplete } from "@material-ui/lab";
 
 import { useMutation } from "@apollo/client";
-import {
-  PROJECT_UPDATE_SPONSOR,
-  PROJECT_UPDATE_LEAD,
-} from "../../../../queries/project";
-
-const getUpdateMutation = (entityName) => {
-  if (entityName === "Sponsor") {
-    return PROJECT_UPDATE_SPONSOR;
-  }
-  if (entityName === "Lead") {
-    return PROJECT_UPDATE_LEAD;
-  }
-};
 
 /**
- * ProjectSummaryStatusUpdate Component
+ * ProjectSummaryAutocomplete Component
+ * @param {String} fieldName - The name of the field to be displayed
+ * @param {String} idColumnName - The name of the id column to be used in the mutation
+ * @param {String} nameColumnName - The name of the name column to be displayed
```

Comment:
i was going to try to suggest a different name for this variable since I couldn't tell what the name column is, but I can't think of anything better.

Reply:
thanks @frankhereford! went with your suggestion